package-dry.json

```
@@ -1,14 +1,14 @@
 {
   "name": "@io.jesi/parent",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Bump minor when updating managed dependency in parent project
See https://github.com/jesims/docs/blob/master/Versioning.md#bumping
```suggestion
  "version": "0.3.0",
```